src/main/java/com/team3/central/config/SecurityConfig.java

```diff
@@ -28,9 +28,9 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
         .cors()
         .disable()
         .authorizeHttpRequests()
-          .antMatchers("/organizer/**","/swagger-ui/**","/v3/api-docs/**")
+          .antMatchers("/organizer/**","/swagger-ui/**","/v3/api-docs/**", "/categories")
             .permitAll()
-        .antMatchers(HttpMethod.GET ,"/events","/events/{id}","/events/getByCategory")
+        .antMatchers(HttpMethod.GET ,"/events","/events/{id}","/events/getByCategory", "/categories")
```

Comment:
First antMatcher is used to set permission for unatuthenticated usert to execute all http methods, second one is for GET only. Choose one please